ci/jenkins/jobs/projects-cloud.yaml

```diff
@@ -1206,8 +1206,127 @@
                     fingerprint: false
                     only-if-success: false
       - '{name}-{test_name}-for-pull-request':
-          test_name: kind-aws-ipv6-only-networkpolicy
-          node: 'antrea-kind-test'
+          test_name: kind-cloud-e2e
+          node: 'antrea-kind-testbed'
+          description: 'This is for running e2e tests on kind.'
+          branches:
+          - ${{sha1}}
+          builders:
+            - builder-e2e-kind:
+               ip_version: 'ipv4'
+               kind_cluster_name: '{test_name}'
+          trigger_phrase: ^(?!Thanks for your PR).*/test-kind-cloud-(e2e|all).*
+          white_list_target_branches: []
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{antrea_admin_list}'
+          org_list: '{antrea_org_list}'
+          white_list: '{antrea_white_list}'
+          only_trigger_phrase: true
+          trigger_permit_all: true
+          throttle_concurrent_builds_enabled: 'true'
+          status_context: jenkins-kind-e2e
+          status_url: --none--
+          success_status: Build finished.
+          failure_status: Failed. Add comment /test-kind-cloud-e2e to re-trigger.
+          error_status: Failed. Add comment /test-kind-cloud-e2e to re-trigger.
+          triggered_status: null
+          started_status: null
+          wrappers:
+          - timeout:
+              fail: true
+              timeout: 135
+              type: absolute
+          publishers:
+          - archive:
+              allow-empty: true
+              artifacts: antrea-test-logs.tar.gz
+              case-sensitive: true
+              default-excludes: true
+              fingerprint: false
+              only-if-success: false
+      - '{name}-{test_name}-for-pull-request':
+          test_name: kind-cloud-conformance
```

Comment:
Please also support all-features-conformance and whole-confomrance for kind job, you can refer to existing CAPV jobs in https://github.com/antrea-io/antrea/blob/1dad03b221ae51e920c7f1ef1d496f35834de3af/ci/jenkins/jobs/projects-cloud.yaml#L286 and https://github.com/antrea-io/antrea/blob/1dad03b221ae51e920c7f1ef1d496f35834de3af/ci/jenkins/jobs/projects-cloud.yaml#L364. After migration we won't be able to use CAPV jobs.